### DISCUSSION TITLE
If no stream , make IMAGE

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6336] [Ideas]
i see the stream makes m3u8 file  , if no stream , how to make automatic m3u8 file with an image , so that it broadcast the image instead . and one the stream is back , the m3u8 file of the stream will be replaced automatic

**Comment by lastpeony** [comment]
Please take a look at this discussion
https://github.com/orgs/ant-media/discussions/6234
I think it covers your case.

**Comment by brixxnatt** [comment]
no the case is the m3u8 file , not the html player

**Comment by lastpeony** [comment]
I couldnt get your point, ıf you explain what you want to achieve as end result I can suggest something

**Comment by brixxnatt** [comment]
so we have m3u8 file when we do the live stream , and when the live stream is off the AMS delete the .ts files and the m3u8 file , so i need to put an m3u8 file in that folder with a video loop to say this channel is offline , then if we go live , the new live m3u8 file will be replaced with the offline one .

**Comment by lastpeony** [comment]
An easy way to manage this is to have a local video for offline playback and use JavaScript to switch between the offline video and the live stream. If the live stream is offline, the local video will play in the video player. When the live stream is online, the offline player will be removed and replaced with the AMS embedded player showing the live stream.
There can be other alternatives like using 2 streams but i think it should be manageable with first way

**Comment by brixxnatt** [comment]
My friend , I am not talking about the video player at all , i dont use it , I use the m3u8 file for some apps , so the problem is not the html player or js player at all , I dont even use those , i am talking about the m3u8 file

**Comment by zerostack0** [comment]
Hi @lastpeony,
I think what @brixxnatt needs is adding new ts files that contain images so the stream can be up 24/7. And whenever there is an active stream, ts files can be these frames. I think @brixxnatt use some tools like vlc to play the m3u8 file and there is no other way to add image. I'm not sure if it's possible right now or not. @lastpeony what do you think about it?
